api/envoy/extensions/filters/http/jwt_authn/v3/README.md

```diff
@@ -30,6 +30,18 @@ If a custom location is desired, `from_headers` or `from_params` can be used to
 
 If a JWT is valid, its payload will be passed to the backend in a new HTTP header specified in `forward_payload_header` field. Its value is base64url-encoded JWT payload in JSON.
 
+## Copy Validated JWT Claims to HTTP Request Headers
```

Comment:
@qiwzhang why is this parallel README.md doc being maintain to the Envoy RST docs? Seems it would be good to converge them for single source-of-truth.

Reply:
OK,  it is a good idea.  I will do that in a separate pr.